Set an absolute threshold to 7 days

tmvumbi2 opened the pull request:
Update the 2 panels "Time since OpenHIM last received transactions from client": change the query to calculate the number of days since the last transaction was received by clients; and change the threshold configuration to highlight in red all clients that haven't sent transactions for 7 days or more

tmvumbi2 committed:
Set an absolute threshold to 7 days
Update the 2 panels "Time since OpenHIM last received transactions from client": change the query to calculate the number of days since the last transaction was received by clients; and change the threshold configuration to highlight in red all clients that haven't sent transactions for 7 days or more

rcrichton commented on grafana/dashboards/applications/rw-dashboard.json:
```diff
@@ -200,7 +200,7 @@
           "editorMode": "code",
           "exemplar": false,
-          "expr": "sort(max_over_time(timestamp(sum by(client) (changes(openhim_transactions_total{client!=\"undefined\"}[5m])) > 0)[$__range:1m]) * 1000)",
+          "expr": "(time() * 1000 - sort(max_over_time(timestamp(sum by(client) (changes(openhim_transactions_total{client!=\"undefined\"}[5m])) > 0)[$__range:1m]) * 1000)) / (1000*60*60*24)",
```
It looks like you might be able to `* 1000` since it appears on both sides of the equation which might make it simpler. I.e.

```suggestion
          "expr": "(time() - sort(max_over_time(timestamp(sum by(client) (changes(openhim_transactions_total{client!=\"undefined\"}[5m])) > 0)[$__range:1m]) )) / (60*60*24)",
```

Note: I haven't tested this as I don't have data so please check it.

tmvumbi2 replied:
Thanks @rcrichton , that's a nice improvement. I pushed the change; and also did test with data, all good

rcrichton commented on grafana/dashboards/applications/rw-dashboard.json:
```diff
@@ -295,7 +295,7 @@
           "editorMode": "code",
           "exemplar": false,
-          "expr": "sort(max_over_time(timestamp(sum by(client) (changes(openhim_transactions_total{client!=\"undefined\"}[5m])) > 0)[$__range:1m]) * 1000)",
+          "expr": "(time() * 1000 - sort(max_over_time(timestamp(sum by(client) (changes(openhim_transactions_total{client!=\"undefined\"}[5m])) > 0)[$__range:1m]) * 1000)) / (1000*60*60*24)",
```
```suggestion
          "expr": "(time() - sort(max_over_time(timestamp(sum by(client) (changes(openhim_transactions_total{client!=\"undefined\"}[5m])) > 0)[$__range:1m]))) / (60*60*24)",
```

rcrichton reviewed (approved):
Looks good to me. I didn't test this as I don't have data. Let me know if you need it tested else visually it looks good barring minor suggestions.

tmvumbi2 committed:
Simplify the PromQL query
The old version is multiplying and dividing by 1000, which can be removed to simplify the query
